src/AppBundle/Resources/views/profile.html.twig

```diff
@@ -14,34 +14,56 @@
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
         </button>
-        <a class="navbar-brand" href="{{ path('fos_user_profile_show') }}">{% trans %}My account{% endtrans %}</a>
+        {# NB: ROLE_ADMIN is also ROLE_COURIER (couriers are admins ;)) #}
+        {% if is_granted('ROLE_COURIER') %}
+          <a class="navbar-brand" href="{{ path('profile_courier_deliveries') }}">{% trans %}My deliveries{% endtrans %}</a>
+        {% elseif is_granted('ROLE_STORE') %}
+          <a class="navbar-brand" href="{{ path('profile_stores') }}">{% trans %}My stores{% endtrans %}</a>
+        {% elseif is_granted('ROLE_RESTAURANT') %}
+          <a class="navbar-brand" href="{{ path('profile_restaurants') }}">{% trans %}My restaurants{% endtrans %}</a>
+        {% else %}
+          <a class="navbar-brand" href="{{ path('fos_user_profile_show') }}">{% trans %}My account{% endtrans %}</a>
+        {% endif %}
       </div>
       <div class="collapse navbar-collapse" id="profile-navbar">
         <ul class="nav navbar-nav navbar-right">
-          <li><a href="{{ path('profile_orders') }}"><i class="fa fa-cube"></i>  {% trans %}My orders{% endtrans %}</a></li>
-          <li><a href="{{ path('profile_addresses') }}"><i class="fa fa-map-marker"></i>  {% trans %}My addresses{% endtrans %}</a></li>
+
           {% if is_granted('ROLE_COURIER') %}
-          <li><a href="{{ path('profile_tasks') }}"><i class="fa fa-calendar-check-o"></i>  {% trans %}My tasks{% endtrans %}</a></li>
-          <li><a href="{{ path('profile_tracking') }}"><i class="fa fa-location-arrow"></i>  {% trans %}My tracking{% endtrans %}</a></li>
+            <li><a href="{{ path('profile_tasks') }}"><i class="fa fa-calendar-check-o"></i>  {% trans %}My tasks{% endtrans %}</a></li>
+            <li><a href="{{ path('profile_tracking') }}"><i class="fa fa-location-arrow"></i>  {% trans %}My tracking{% endtrans %}</a></li>
           {% endif %}
-          {% if is_granted('ROLE_STORE') %}
-          <li><a href="{{ path('profile_stores') }}"><i class="fa fa-shopping-cart"></i>  {% trans %}My stores{% endtrans %}</a></li>
+
+          {% if is_granted('ROLE_STORE') and is_granted('ROLE_COURIER') %}
```

Comment:
`and is_granted('ROLE_COURIER')` is useless.

Reply:
It's a little bit convoluted (i.e. ugly), but if the user has `ROLE_STORE` only, `profile_stores` will be accessible on the main link on the left of the navbar.
Only if he's `COURIER` and `STORE` we need to display `profile_stores` link here.